csi/src/node.rs

```diff
@@ -33,6 +31,11 @@ pub struct Node {
     pub filesystems: Vec<String>,
 }
 
+// Timeout settings for device attach
+// 10 retries at 100ms intervals
```

Comment:
comment seems out of date. I'd say just remove it since the variable names are good.

Reply:
Done.